test/modules/core/webgl/texture.spec.js

```diff
@@ -34,106 +34,111 @@ test('WebGL#Texture2D construct/delete', t => {
   t.end();
 });
 
+function isFormatSupported(format, glContext) {
+  format = Number(format);
+  const opts = Object.assign({format}, TEXTURE_FORMATS[format]);
+  if (!Texture2D.isSupported(glContext, {format}) || (!isWebGL2(glContext) && opts.compressed)) {
+    return false;
+  }
+  return true;
+}
 test('WebGL#Texture2D check formats', t => {
-  const {gl} = fixture;
+  const {gl, gl2} = fixture;
 
   const WEBGL1_FORMATS = [GL.RGB, GL.RGBA, GL.LUMINANCE_ALPHA, GL.LUMINANCE, GL.ALPHA];
+  const WEBGL2_FORMATS = [GL.R32F, GL.RG32F, GL.RGB32F, GL.RGBA32F];
```

Comment:
One of these should be available under WebGL1 with an extension, there is a luma FEATURE to detect it. 

Could be worth testing.